packages/manager/modules/manager-pci-common/src/api/hook/usePCIFeatureAvailability.ts

```diff
@@ -0,0 +1,30 @@
+import { useQuery } from '@tanstack/react-query';
+import { apiClient } from '@ovh-ux/manager-core-api';
+import { useApplicationName } from '@/hooks/useApplicationName/useApplicationName';
+
+type FeatureAvailabilityQueryParams = Partial<{
+  app: string;
+}>;
+
+const getFeatureAvailability = (
+  features: string[],
+  params?: FeatureAvailabilityQueryParams,
+) =>
+  apiClient.aapi.get(`/feature/${features.join(',')}/availability`, {
+    params,
+  });
+
+const useFeatureAvailability = <T extends string[]>(
+  features: T,
+  params?: FeatureAvailabilityQueryParams,
+) =>
+  useQuery({
+    queryKey: ['feature-availability', ...features, params],
+    queryFn: () => getFeatureAvailability(features, params),
+    select: ({ data }) => new Map(features.map((f) => [f, data[f] ?? false])),
```

Comment:
We could prevent `data[f]` to be `any`:

```suggestion
    select: ({ data }: { data: Record<T[number], boolean> })) => new Map(features.map((f) => [f, data[f] ?? false])),
```

Reply:
data isn't any, the type comes from getFeatureAvailability

Reply:
Here, `getFeatureAvailability` returns a `Promise<AxiosResponse<any, any>>